components/search_engines/brave_prepopulated_engines.cc

```diff
@@ -0,0 +1,372 @@
+  NULL,
+  NULL,
+  NULL,
+  NULL,
+  NULL,
+  NULL,
+  NULL,
+  NULL,
+  NULL,
+  0,
+  SEARCH_ENGINE_OTHER,
+  PREPOPULATED_ENGINE_ID_QWANT,
+};
+
+const PrepopulatedEngine searx = {
+  L"searx",
+  L":x",
+  "https://searx.me/favicon.ico",
+  "https://searx.me/?q={searchTerms}&categories=general",
+  "UTF-8",
+  NULL,
+  NULL,
+  NULL,
+  NULL,
+  NULL,
+  NULL,
+  NULL,
+  NULL,
+  NULL,
+  NULL,
+  0,
+  SEARCH_ENGINE_OTHER,
+  PREPOPULATED_ENGINE_ID_SEARX,
+};
+
+const PrepopulatedEngine semanticscholar = {
+  L"Semantic Scholar",
+  L":ss",
+  "https://www.semanticscholar.org/img/favicon.png",
+  "https://www.semanticscholar.org/search?q={searchTerms}",
+  "UTF-8",
+  NULL,
+  NULL,
+  NULL,
+  NULL,
+  NULL,
+  NULL,
+  NULL,
+  NULL,
+  NULL,
+  NULL,
+  0,
+  SEARCH_ENGINE_OTHER,
+  PREPOPULATED_ENGINE_ID_SEMANTICSCHOLAR,
+};
+
+const PrepopulatedEngine stackoverflow = {
+  L"Stack Overflow",
+  L":s",
+  "https://cdn.sstatic.net/Sites/stackoverflow/img/favicon.ico",
+  "https://stackoverflow.com/search?q={searchTerms}",
+  "UTF-8",
+  NULL,
+  NULL,
+  NULL,
+  NULL,
+  NULL,
+  NULL,
+  NULL,
+  NULL,
+  NULL,
+  NULL,
+  0,
+  SEARCH_ENGINE_OTHER,
+  PREPOPULATED_ENGINE_ID_STACKOVERFLOW,
+};
+
+const PrepopulatedEngine startpage = {
+  L"StartPage",
+  L":sp",
+  "https://www.startpage.com/favicon.ico",
+  "https://www.startpage.com/do/dsearch?query={searchTerms}&cat=web&pl=opensearch",
+  "UTF-8",
+  "https://www.startpage.com/cgi-bin/csuggest?query={searchTerms}&limit=10&format=json",
+  NULL,
+  NULL,
+  NULL,
+  NULL,
+  NULL,
+  NULL,
+  NULL,
+  NULL,
+  NULL,
+  0,
+  SEARCH_ENGINE_OTHER,
+  PREPOPULATED_ENGINE_ID_STARTPAGE,
+};
+
+const PrepopulatedEngine twitter = {
+  L"Twitter",
+  L":t",
+  "https://twitter.com/favicon.ico",
+  "https://twitter.com/search?q={searchTerms}&source=desktop-search",
+  "UTF-8",
+  NULL,
+  NULL,
+  NULL,
+  NULL,
+  NULL,
+  NULL,
+  NULL,
+  NULL,
+  NULL,
+  NULL,
+  0,
+  SEARCH_ENGINE_OTHER,
+  PREPOPULATED_ENGINE_ID_TWITTER,
+};
+
+const PrepopulatedEngine wikipedia = {
+  L"Wikipedia",
+  L":w",
+  "https://en.wikipedia.org/favicon.ico",
+  "https://en.wikipedia.org/wiki/Special:Search?search={searchTerms}",
+  "UTF-8",
+  NULL,
+  NULL,
+  NULL,
+  NULL,
+  NULL,
+  NULL,
+  NULL,
+  NULL,
+  NULL,
+  NULL,
+  0,
+  SEARCH_ENGINE_OTHER,
+  PREPOPULATED_ENGINE_ID_WIKIPEDIA,
+};
+
+const PrepopulatedEngine wolframalpha = {
+  L"Wolfram Alpha",
+  L":wa",
+  "https://www.wolframalpha.com/favicon.ico",
+  "https://www.wolframalpha.com/input/?i={searchTerms}",
+  "UTF-8",
+  NULL,
+  NULL,
+  NULL,
+  NULL,
+  NULL,
+  NULL,
+  NULL,
+  NULL,
+  NULL,
+  NULL,
+  0,
+  SEARCH_ENGINE_OTHER,
+  PREPOPULATED_ENGINE_ID_WOLFRAMALPHA,
+};
+
+const PrepopulatedEngine yandex = {
+  L"Yandex",
+  L":ya",
+  "https://yandex.st/lego/_/rBTjd6UOPk5913OSn5ZQVYMTQWQ.ico",
+  "https://yandex.com/search/?text={searchTerms}&clid=2274777",

```

Comment:
looks like originally clid was set by client OS? https://github.com/brave/browser-laptop/pull/5964/files#diff-1e3d00f00cc34a0805abaafa53c7d7a0R134

i'm ok with setting it to be a constant

Reply:
I copied this directly from https://github.com/brave/browser-laptop/blob/master/js/data/searchProviders.js which doesn't appear to have "platformClientId" any more.

Reply:
Good find, @diracdeltas - I forgot about that... I wonder if that regressed on purpose? Gonna take a peek

Reply:
OK interesting- it was removed by @bridiver via https://github.com/brave/browser-laptop/commit/c001b8b9cbc45bb39af28777a6472dca8a008640#diff-1e3d00f00cc34a0805abaafa53c7d7a0 during a refactor.  I'll double check that we don't want platform ID and report back

Reply:
update: we are good to go 😄 we don't need to pass `platformClientId` 👍